linera-storage/src/db_storage.rs

```diff
@@ -224,37 +225,55 @@ pub mod metrics {
     });
 }
 
+/// The default key used when the root_key contains the information.
+pub(crate) const DEFAULT_KEY: &[u8] = &[0];
+
+/// The second key used when the root_key contains the information.
+pub(crate) const ONE_KEY: &[u8] = &[1];
```

Comment:
Maybe it would be cleaner to create separate constants, even if some of them have the same value: `BLOCK_KEY`, `LITE_CERTIFICATE_KEY`, `BLOB_KEY`, `BLOB_STATE_KEY`, etc.